packages/waku/src/router/define-router.tsx

```diff
@@ -565,7 +565,11 @@ export function unstable_defineRouter(fns: {
     }
     const pathConfigItem = await getPathConfigItem(input.pathname);
     if (pathConfigItem?.type === 'api' && fns.handleApi) {
-      return fns.handleApi(input.req);
+      const req = new Request(
+        new URL(input.pathname, input.req.url),
+        input.req,
+      );
+      return fns.handleApi(req);
```

Comment:
This also looks good to me. CI failed because this drops query parameter. It needs to preserved like

```js
const url = new URL(input.req.url)
url.pathname = input.pathname;
const req = new Request(url, input.req)
```